src/code/z_sub_s.c

```diff
@@ -260,9 +264,13 @@ s32 SubS_ChangeAnimationByInfoS(SkelAnime* skelAnime, AnimationInfoS* animations
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013D83C.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013D8DC.s")
+s8 SubS_IsObjectLoaded(s8 index, GlobalContext* globalCtx) {
+    return !Object_IsLoaded(&globalCtx->objectCtx, index) ? false : true;
+}
```

Comment:
This might me the most pointless function in the entire codebase... I don't suppose some variation on `!!` works (`!!!`?)

Reply:
Nope.